Source/Scene/Model.js

```diff
@@ -173,6 +172,8 @@ var uriToGuid = {};
  * {@link https://github.com/KhronosGroup/glTF/blob/master/extensions/2.0/Khronos/KHR_techniques_webgl/README.md|KHR_techniques_webgl}
  * </li><li>
  * {@link https://github.com/KhronosGroup/glTF/blob/master/extensions/2.0/Khronos/KHR_texture_transform/README.md|KHR_texture_transform}
+ * </li><li>
+ * {@link https://github.com/KhronosGroup/glTF/pull/1751|KHR_texture_basisu}
```

Comment:
As of this writing the PR has been accepted: https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_texture_basisu

Reply:
I closed https://github.com/CesiumGS/cesium/issues/9054, we can get the fix into this PR now.